services/web/server/requirements/_base.in

```diff
@@ -17,67 +17,42 @@
 --requirement ../../../../packages/service-library/requirements/_aiohttp.in
 
 
-# aiohttp + extensions
-aiohttp
-aiohttp_jinja2
-aiohttp_session[secure]
-aiohttp_security
-aiohttp-swagger[performance]
-gunicorn[setproctitle]
-
-
 # web-sockets
 # From 5.0.0, https://github.com/miguelgrinberg/python-socketio/blob/main/CHANGES.md
 # test_resource_manager.py::test_websocket_resource_management fails because
 # socket_id saved in redis does not correspond to client's sio
 python-socketio~=4.6.1
 
-# postgres db
-aiopg[sa]
-asyncpg
-
-# i/o
-aiofiles
-
-# redis
-aioredis
-aioredlock
-
-# RabbitMQ client
-aio-pika
-
-# email
-aiosmtplib
-jinja_app_loader
-
-
-# data models
-pydantic[email]
-
-# security
-cryptography
-passlib
-
-# json
-orjson
-jsondiff
-json2html
-
-# asyncio debug
-aiodebug
+# SEE services/web/server/tests/unit/isolated/test_utils.py::test_yarl_url_compose_changed_with_latest_release
+yarl<1.6
```

Comment:
Could you add when this could be unpinned? I cannot find the reference from the comment.

Reply:
what do you mean? I think the line 26 is pretty clear, ritgh? This is the test

https://github.com/ITISFoundation/osparc-simcore/blob/6939ced72fbcaf42ecc5b0870395fbcc08bffd4d/services/web/server/tests/unit/isolated/test_utils.py#L54